### PR TITLE
inventory: fix black screen frames when switching inventory

### LINKED ISSUE #75: OG bugfix: blinking when switching between inventory modes
When navigating from the main menu ring to the keys or the options or back, the game renders a black screen briefly.

### PR DESCRIPTION
**Pull request by walkawayy**
Resolves #75.

### CODE REVIEW

**Comment by walkawayy** [comment]
Cleaned continuation of #315 since that approach was reverted.

File: src/game/output.h [hunk 1]
```diff
@@ -74,4 +75,4 @@ void Output_FadeToBlack();
 
 void Output_ApplyWaterEffect(float *r, float *g, float *b);
 
-bool Output_MakeScreenshot(const char *path);
+bool Output_MakeScreenshot(const char *path);
```
Comment (rr-) [inline src/game/output.h:78]: Missing newlines :)

File: src/gfx/3d/3d_renderer.h [hunk 2]
```diff
@@ -52,3 +52,4 @@ void GFX_3D_Renderer_SetBlendingEnabled(
     GFX_3D_Renderer *renderer, bool is_enabled);
 void GFX_3D_Renderer_SetTexturingEnabled(
     GFX_3D_Renderer *renderer, bool is_enabled);
+void GFX_3D_Renderer_RenderEmpty();
```
Comment (rr-) [inline src/gfx/3d/3d_renderer.h:55]: Missing newline.

**Review by rr- — APPROVED** [review]
LGTM apart from NL near EOF.

**Comment by walkawayy** [comment]
Oof I keep doing that. Only noticed the missing one in the .c file. Thanks.

**Review by rr- — APPROVED** [review]

**Comment by rr-** [comment]
Thank you for excellent analysis why this happens and the fix.

**Pull request merged**